file.go

```diff
@@ -246,3 +246,24 @@ func (db *DbFile) Dumpdb() error {
 
 	return err
 }
+
+// VisitTableRecords performs an inorder traversal of all cells in the
+// btree for the table with the given name, passing the (optional,
+// hence nullable) RowID, and record-decoded payload of each cell to
+// the visitor function `f`.
+func (db *DbFile) VisitTableRecords(tableName string, f func(*int64, Record) error) error {
+	for _, table := range db.tables {
+		if table.name == tableName {
```

Comment:
perhaps inverse the condition and `continue`, so the code isn't pushed too far on the right?

Reply:
done